frontend/packages/dev-console/src/components/pipelineruns/logs/logs-utils.ts

```diff
@@ -97,7 +97,7 @@ export const getDownloadAllLogsCallback = (
       const allStepUrls = steps.reduce((stepUrls, currentStep) => {
         const { name } = currentStep;
         const currentStatus = containerToLogSourceStatus(currentStep);
-        if (currentStatus === LOG_SOURCE_WAITING) return stepUrls;
+        if ([LOG_SOURCE_WAITING, undefined].includes(currentStatus)) return stepUrls;
```

Comment:
Based on the other comment
```suggestion
        if ([LOG_SOURCE_WAITING, LOG_SOURCE_NOT_READY].includes(currentStatus)) return stepUrls;
```